packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -15,6 +15,11 @@ export type ProgressSlots = {
  * Progress Props
  */
 export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
+  /**
+   * The appearance of the bar and track.
+   * @default 'rounded'
+   */
+  appearance?: 'rounded' | 'rectangular';
```

Comment:
FWIW I prefer calling this prop `shape` instead of `appearance`. We usually use `appearance` for things that affect the colors used, though we don't have any strict rules. It might be worth reserving `appearance` for future use in case we add color variants to Progress.

Reply:
`shape` makes a lot of sense! I've made the changes in the PR